cdk-testsuite/tests/openshiftSanity.py

```diff
@@ -0,0 +1,269 @@
+'''
+Created on Jun 29, 2016
+
+@author: amit
+'''
+from avocado import Test
+from avocado import VERSION
+import imp
+import logging
+import os
+import platform
+import re
+
+log = logging.getLogger("Openshift.Debug")
+
+def new_project(self, port, username, password, projectname, registry, servicename, tempalte = False, dbservicename = "default"):
+    '''
+    Adding a new application to the project and returns output of the oc command executed
+    Args:
+        self (object): Object of the current method
+        username (string): username of openshift web console to be used
+        password (string): password of openshift web console to be used
+        projectname (string): name of the project to be added to the openshift server
+        registry (string): registry path/location
+        servicename (string): name of the service to be exposed outside
+        tempalte (boolean): if creating the s2i using openshift template then it takes value as True otherwise the default value False 
+        dbservicename (string): Takes the database service name if specified otherwise default
+    '''
+    output = openshift.oc_usr_login(self, port, username, password)
+    self.assertIn("Login successful", output, "Login failed")
+        
+    output = openshift.add_new_project(self, projectname)
+    self.assertIn(projectname, output, "Failed to create " +projectname)
+    
+    if not tempalte:
+        output = openshift.add_new_app(self, registry)
```

Comment:
The below lines of code are repetitive, just with the change of a variable. Better to have this in a separate function and just call it here.